drivers/dma/Kconfig.stm32

```diff
@@ -27,10 +27,11 @@ config DMA_STM32_V2
 
 config DMAMUX_STM32
 	bool
+	default y
```

Comment:
It seems that 	`config DMA_STM32_V1` and `config DMA_STM32_V2` also requires  **default y**

Reply:
When compiled the tests/drivers/spi/spi_loopback with _nucleo_l4r5zi.conf_ where `CONFIG_DMA=y`

Reply:
`DMA_STM32_V1` and `DMA_STM32_V2` set to `y` from the soc configuration. I see that #31163 takes care of that.

Reply:
ok with PR #31163